### ISSUE TITLE
Banks and Stock Exchanges not calculated

### ISSUE THREAD

**Issue by TerryDad2** [Wahazar/augmented2#7]
I am finding that banks and stock exchange bonuses are not being added to trade. The stock exchange is recognized with superhighways.
![image](https://user-images.githubusercontent.com/75283838/100679997-839f6500-3325-11eb-8571-c0fef8c6a3cc.png)

**Comment by Wahazar** [comment]
Yes, it is indented, banks increase gold tax, not overall trade. In conventional freeciv banks also didn't increase trade (just gold and luxury)
